data/data/bootstrap/baremetal/files/etc/containers/systemd/metal3-baremetal-operator.container.template

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Metal3 deployment service
+BindsTo=ironic-volume.service
+Requires=build-metal3-env.service wait-iptables-init.service image-customization.service build-ironic-env.service
+Wants=network-online.target ironic-httpd.service ironic-inspector.service ironic-ramdisk-logs.service
+After=network-online.target ironic-volume.service build-ironic-env.service wait-iptables-init.service provisioning-interface.service ironic-dnsmasq.service image-customization.service
+
+[Container]
+ContainerName=baremetal-operator
+Image=$METAL3_BAREMETAL_OPERATOR_IMAGE
+Exec=/baremetal-operator -build-preprov-image
+Network=host
+Volume=${AUTH_DIR}:/auth:z,ro
+Volume=/opt/openshift:/opt/openshift:z,ro
+Volume=ironic.volume:/shared:z
```

Comment:
This shouldn't be required for BMO.

Reply:
Yep, BMO does not use the /shared location

Reply:
Done